ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml

```diff
@@ -23,12 +23,24 @@ parameters:
 - name: DEPLOY_VERSION
   value: "v3.11"
 - name: TEST_IMAGE
-  value: "registry.svc.ci.openshift.org/azure/test-base:latest"
+  value: quay.io/kwoodson/bushslicer
+  #value: "registry.svc.ci.openshift.org/azure/test-base:latest"
 - name: TEST_COMMAND
-  value: "make e2e"
+  value: "/usr/bin/scl enable rh-git29 rh-ror50 -- cucumber features/cli/build.feature:5"
```

Comment:
This too

Reply:
same as above.  Any suggestion on how to test this with different environment variables?  Can I set them on the CLI and have them run?

Reply:
Yes, if you run ci-operator locally, exposing them in your env is the same thing.

Reply:
Thanks, will give that a shot.